### PR TITLE
Added pre-trained weights for ShuffleNetV2 x1.5 and x2.0

### LINKED ISSUE #524: x1.5 model consistently underperforming
I'm using NanoDet-Plus for research purposes but ran into a weird issue where the x1.5 model variants consistently underperform compared to the x1.0 models. This happens on [VISEM](https://paperswithcode.com/dataset/visem-tracking), [Argoverse-HD](https://paperswithcode.com/dataset/argoverse-hd), and even COCO2017.
For COCO2017 I used the stock config provided in the git repo ([x1.0](https://github.com/RangiLyu/nanodet/blob/main/config/nanodet-plus-m_320.yml) and [x1.5](https://github.com/RangiLyu/nanodet/blob/main/config/nanodet-plus-m-1.5x_320.yml)), but the COCO mAP already separates the two after the first 10 epochs, with the bigger variant consistently scoring lower. This holds true for all three datasets from my observations:
image

I get the following mAP metrics:

|   |x1.0|x1.5|epochs|
|---|---|---|---|
|VISEM|6.9%|2.0%|100|
|Argoverse-HD|24.1%|21.2%|70|
|COCO2017|20.9%|14.5%|40*|

*The models are still training as of now, but the separation in mAP mentioned above is already distinctly noticeable in the logs. I will let the runs continue until 300 epochs have been reached, as the stock config dictates.

Here are the configs I used for VISEM and Argoverse-HD:
<details>
<summary>VISEM x1.0</summary>

```yml
save_dir: workspace/baseline/visem/nanodet-plus-m-1.0x-dgxa100
model:
  arch:
    backbone:
      name: ShuffleNetV2
      model_size: 1.0x
      out_stages: [2, 3, 4]
      activation: LeakyReLU
      channels: 3
    fpn:
      name: GhostPAN
      in_channels: [116, 232, 464]
      out_channels: 96
      kernel_size: 5
      num_extra_level: 1
      use_depthwise: true
      activation: LeakyReLU
    head:
      name: NanoDetPlusHead
      num_classes: 3
      input_channel: 96
      feat_channels: 96
      stacked_convs: 2
      kernel_size: 5
      strides: [8, 16, 32, 64]
      activation: LeakyReLU
      reg_max: 7
      norm_cfg: {type: BN}
      loss:
        loss_qfl: {name: QualityFocalLoss, use_sigmoid: true, beta: 2.0, loss_weight: 1.0}
        loss_dfl: {name: DistributionFocalLoss, loss_weight: 0.25}
        loss_bbox: {name: GIoULoss, loss_weight: 2.0}
    name: NanoDetPlus
    detach_epoch: 10
    aux_head:
      name: SimpleConvHead
      num_classes: 3
      input_channel: 192
      feat_channels: 192
      stacked_convs: 4
      strides: [8, 16, 32, 64]
      activation: LeakyReLU
      reg_max: 7
  weight_averager: {name: ExpMovingAverager, decay: 0.9998}
device:
  precision: 16
  gpu_ids: [0]
  workers_per_gpu: 28
  batchsize_per_gpu: 92
schedule:
  optimizer: {name: AdamW, lr: 0.001, weight_decay: 1.0e-05}
  warmup: {name: linear, steps: 500, ratio: 0.0001}
  total_epochs: 10
  lr_schedule: {name: CosineAnnealingLR, T_max: 300, eta_min: 5.0e-05}
  val_intervals: 10
log: {interval: 50}
test: {}
grad_clip: 35
evaluator: {name: CocoDetectionEvaluator, save_key: mAP}
class_names: &id001 [sperm, cluster, small/pinhead]
data:
  train:
    name: VISEMDataset
    pipeline:
      perspective: 0
      scale: [0.8, 1.2]
      stretch:
      - [0.95, 1.05]
      - [0.95, 1.05]
      rotation: 180
      shear: 10
      translate: 0.2
      flip: 0.5
      brightness: 0.2
      contrast: [0.6, 1.4]
      saturation: [0.6, 1.2]
      normalize: &id002
      - [123.675, 116.28, 103.53]
      - [58.395, 57.12, 57.375]
    img_path: ../data/VISEM/VISEM_Tracking_Train_v4/Train
    ann_path: ../data/VISEM/VISEM_Tracking_Train_v4/Train
    class_names: *id001
    input_size: &id003 [640, 480]
    keep_ratio: false
  val:
    name: VISEMDataset
    pipeline:
      normalize: *id002
    img_path: ../data/VISEM/VISEM_Tracking_Train_v4/Train
    ann_path: ../data/VISEM/VISEM_Tracking_Train_v4/Train
    class_names: *id001
    input_size: *id003
    keep_ratio: false
``` 

</details>

<details>
<summary>VISEM x1.5</summary>

```yml
save_dir: workspace/baseline/visem/nanodet-plus-m-1.5x-dgxa100
model:
  arch:
    backbone:
      name: ShuffleNetV2
      model_size: 1.5x
      out_stages: [2, 3, 4]
      activation: LeakyReLU
      channels: 3
    fpn:
      name: GhostPAN
      in_channels: [176, 352, 704]
      out_channels: 128
      kernel_size: 5
      num_extra_level: 1
      use_depthwise: true
      activation: LeakyReLU
    head:
      name: NanoDetPlusHead
      num_classes: 3
      input_channel: 128
      feat_channels: 128
      stacked_convs: 2
      kernel_size: 5
      strides: [8, 16, 32, 64]
      activation: LeakyReLU
      reg_max: 7
      norm_cfg: {type: BN}
      loss:
        loss_qfl: {name: QualityFocalLoss, use_sigmoid: true, beta: 2.0, loss_weight: 1.0}
        loss_dfl: {name: DistributionFocalLoss, loss_weight: 0.25}
        loss_bbox: {name: GIoULoss, loss_weight: 2.0}
    name: NanoDetPlus
    detach_epoch: 10
    aux_head:
      name: SimpleConvHead
      num_classes: 3
      input_channel: 256
      feat_channels: 256
      stacked_convs: 4
      strides: [8, 16, 32, 64]
      activation: LeakyReLU
      reg_max: 7
  weight_averager: {name: ExpMovingAverager, decay: 0.9998}
device:
  precision: 16
  gpu_ids: [0]
  workers_per_gpu: 28
  batchsize_per_gpu: 88
schedule:
  optimizer: {name: AdamW, lr: 0.001, weight_decay: 1.0e-05}
  warmup: {name: linear, steps: 500, ratio: 0.0001}
  total_epochs: 10
  lr_schedule: {name: CosineAnnealingLR, T_max: 300, eta_min: 5.0e-05}
  val_intervals: 10
log: {interval: 50}
test: {}
grad_clip: 35
evaluator: {name: CocoDetectionEvaluator, save_key: mAP}
class_names: &id001 [sperm, cluster, small/pinhead]
data:
  train:
    name: VISEMDataset
    pipeline:
      perspective: 0
      scale: [0.8, 1.2]
      stretch:
      - [0.95, 1.05]
      - [0.95, 1.05]
      rotation: 180
      shear: 10
      translate: 0.2
      flip: 0.5
      brightness: 0.2
      contrast: [0.6, 1.4]
      saturation: [0.6, 1.2]
      normalize: &id002
      - [123.675, 116.28, 103.53]
      - [58.395, 57.12, 57.375]
    img_path: ../data/VISEM/VISEM_Tracking_Train_v4/Train
    ann_path: ../data/VISEM/VISEM_Tracking_Train_v4/Train
    class_names: *id001
    input_size: &id003 [640, 480]
    keep_ratio: false
  val:
    name: VISEMDataset
    pipeline:
      normalize: *id002
    img_path: ../data/VISEM/VISEM_Tracking_Train_v4/Train
    ann_path: ../data/VISEM/VISEM_Tracking_Train_v4/Train
    class_names: *id001
    input_size: *id003
    keep_ratio: false
``` 

</details>

<details>
<summary>Argoverse-HD x1.0</summary>

```yml
save_dir: workspace/baseline/argoverse/nanodet-plus-m-1.0x-dgxa100
model:
  arch:
    backbone:
      name: ShuffleNetV2
      model_size: 1.0x
      out_stages: [2, 3, 4]
      activation: LeakyReLU
      channels: 3
    fpn:
      name: GhostPAN
      in_channels: [116, 232, 464]
      out_channels: 96
      kernel_size: 5
      num_extra_level: 1
      use_depthwise: true
      activation: LeakyReLU
    head:
      name: NanoDetPlusHead
      num_classes: 8
      input_channel: 96
      feat_channels: 96
      stacked_convs: 2
      kernel_size: 5
      strides: [8, 16, 32, 64]
      activation: LeakyReLU
      reg_max: 7
      norm_cfg: {type: BN}
      loss:
        loss_qfl: {name: QualityFocalLoss, use_sigmoid: true, beta: 2.0, loss_weight: 1.0}
        loss_dfl: {name: DistributionFocalLoss, loss_weight: 0.25}
        loss_bbox: {name: GIoULoss, loss_weight: 2.0}
    name: NanoDetPlus
    detach_epoch: 10
    aux_head:
      name: SimpleConvHead
      num_classes: 8
      input_channel: 192
      feat_channels: 192
      stacked_convs: 4
      strides: [8, 16, 32, 64]
      activation: LeakyReLU
      reg_max: 7
  weight_averager: {name: ExpMovingAverager, decay: 0.9998}
device:
  precision: 16
  gpu_ids: [0]
  workers_per_gpu: 28
  batchsize_per_gpu: 6
schedule:
  optimizer: {name: AdamW, lr: 0.0003, weight_decay: 0.01}
  warmup: {name: linear, steps: 500, ratio: 0.0001}
  total_epochs: 70
  lr_schedule: {name: CosineAnnealingLR, T_max: 300, eta_min: 5.0e-05}
  val_intervals: 10
log: {interval: 50}
test: {}
grad_clip: 35
evaluator: {name: CocoDetectionEvaluator, save_key: mAP}
class_names: &id001 [person, bicycle, car, motorcycle, bus, truck, traffic_light,
  stop_sign]
data:
  train:
    name: ArgoverseDataset
    pipeline:
      perspective: 0
      scale: [0.8, 1.2]
      stretch:
      - [0.95, 1.05]
      - [0.95, 1.05]
      rotation: 0
      shear: 0
      translate: 0.1
      flip: 0
      brightness: 0.2
      contrast: [0.6, 1.4]
      saturation: [0.6, 1.2]
      normalize: &id002
      - [123.675, 116.28, 103.53]
      - [58.395, 57.12, 57.375]
    class_names: *id001
    input_size: &id003 [1680, 1050]
    keep_ratio: true
    img_path: ../data/Argoverse-1.1/tracking/train
    ann_path: ../data/Argoverse-HD/annotations/train.json
  val:
    name: ArgoverseDataset
    pipeline:
      normalize: *id002
    class_names: *id001
    input_size: *id003
    keep_ratio: true
    img_path: ../data/Argoverse-1.1/tracking/val
    ann_path: ../data/Argoverse-HD/annotations/val.json
``` 

</details>

<details>
<summary>Argoverse-HD x1.5</summary>

```yml
save_dir: workspace/baseline/argoverse/nanodet-plus-m-1.5x-dgxa100
model:
  arch:
    backbone:
      name: ShuffleNetV2
      model_size: 1.5x
      out_stages: [2, 3, 4]
      activation: LeakyReLU
      channels: 3
    fpn:
      name: GhostPAN
      in_channels: [176, 352, 704]
      out_channels: 128
      kernel_size: 5
      num_extra_level: 1
      use_depthwise: true
      activation: LeakyReLU
    head:
      name: NanoDetPlusHead
      num_classes: 8
      input_channel: 128
      feat_channels: 128
      stacked_convs: 2
      kernel_size: 5
      strides: [8, 16, 32, 64]
      activation: LeakyReLU
      reg_max: 7
      norm_cfg: {type: BN}
      loss:
        loss_qfl: {name: QualityFocalLoss, use_sigmoid: true, beta: 2.0, loss_weight: 1.0}
        loss_dfl: {name: DistributionFocalLoss, loss_weight: 0.25}
        loss_bbox: {name: GIoULoss, loss_weight: 2.0}
    name: NanoDetPlus
    detach_epoch: 10
    aux_head:
      name: SimpleConvHead
      num_classes: 8
      input_channel: 256
      feat_channels: 256
      stacked_convs: 4
      strides: [8, 16, 32, 64]
      activation: LeakyReLU
      reg_max: 7
  weight_averager: {name: ExpMovingAverager, decay: 0.9998}
device:
  precision: 16
  gpu_ids: [0]
  workers_per_gpu: 28
  batchsize_per_gpu: 6
schedule:
  optimizer: {name: AdamW, lr: 0.0003, weight_decay: 0.01}
  warmup: {name: linear, steps: 500, ratio: 0.0001}
  total_epochs: 70
  lr_schedule: {name: CosineAnnealingLR, T_max: 300, eta_min: 5.0e-05}
  val_intervals: 10
log: {interval: 50}
test: {}
grad_clip: 35
evaluator: {name: CocoDetectionEvaluator, save_key: mAP}
class_names: &id001 [person, bicycle, car, motorcycle, bus, truck, traffic_light,
  stop_sign]
data:
  train:
    name: ArgoverseDataset
    pipeline:
      perspective: 0
      scale: [0.8, 1.2]
      stretch:
      - [0.95, 1.05]
      - [0.95, 1.05]
      rotation: 0
      shear: 0
      translate: 0.1
      flip: 0
      brightness: 0.2
      contrast: [0.6, 1.4]
      saturation: [0.6, 1.2]
      normalize: &id002
      - [123.675, 116.28, 103.53]
      - [58.395, 57.12, 57.375]
    class_names: *id001
    input_size: &id003 [1680, 1050]
    keep_ratio: true
    img_path: ../data/Argoverse-1.1/tracking/train
    ann_path: ../data/Argoverse-HD/annotations/train.json
  val:
    name: ArgoverseDataset
    pipeline:
      normalize: *id002
    class_names: *id001
    input_size: *id003
    keep_ratio: true
    img_path: ../data/Argoverse-1.1/tracking/val
    ann_path: ../data/Argoverse-HD/annotations/val.json
``` 

</details>

Is this erroneous behavior?  Is there something wrong with my setup?

### PR DESCRIPTION
**Pull request by raember**
[Pre-trained shufflenetv2 (x1.5 and x2.0) not being supported](https://github.com/pytorch/vision/issues/3257) links to [an issue about adding more pre-trained weights](https://github.com/pytorch/vision/issues/2707), which has a link to [a merged PR adding them to the repo](https://github.com/pytorch/vision/pull/5906), which in turn links the missing weights.

Closes #524

### CODE REVIEW

**Comment by RangiLyu** [comment]
Thanks for your contribution!

File: nanodet/model/backbone/shufflenetv2.py [hunk 1]
```diff
@@ -7,8 +7,8 @@
 model_urls = {
     "shufflenetv2_0.5x": "https://download.pytorch.org/models/shufflenetv2_x0.5-f707e7126e.pth",  # noqa: E501
     "shufflenetv2_1.0x": "https://download.pytorch.org/models/shufflenetv2_x1-5666bf0f80.pth",  # noqa: E501
-    "shufflenetv2_1.5x": None,
-    "shufflenetv2_2.0x": None,
+    "shufflenetv2_1.5x": "https://download.pytorch.org/models/shufflenetv2_x1_5-3c479a10.pth",
```
Comment (RangiLyu) [inline nanodet/model/backbone/shufflenetv2.py:10]:
```suggestion
    "shufflenetv2_1.5x": "https://download.pytorch.org/models/shufflenetv2_x1_5-3c479a10.pth",  # noqa: E501
```

File: nanodet/model/backbone/shufflenetv2.py [hunk 2]
```diff
@@ -7,8 +7,8 @@
...
+    "shufflenetv2_2.0x": "https://download.pytorch.org/models/shufflenetv2_x2_0-8be3c8ee.pth",
```
Comment (RangiLyu) [inline nanodet/model/backbone/shufflenetv2.py:11]:
```suggestion
    "shufflenetv2_2.0x": "https://download.pytorch.org/models/shufflenetv2_x2_0-8be3c8ee.pth",  # noqa: E501
```

**Comment by RangiLyu** [comment]
lint failure has nothing to do with this PR. merged
